src/TimeSelect.jsx

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var React = require('react');
+var Input = require('react-bootstrap').Input;
+var ReactIntl = require('react-intl');
+
+var TimePicker = React.createClass({
+  mixins: [ ReactIntl.IntlMixin ],
+
+  propTypes: {
+    className: React.PropTypes.string,
+    label: React.PropTypes.string,
+    name: React.PropTypes.string,
+    value: React.PropTypes.instanceOf(Date),
+    start: React.PropTypes.number,
```

Comment:
there so needs to be a react time prop type validator, passing time as an integer doesn't quite fit.

There is a date validator available so we could spend an amount of time and extend this project: https://www.npmjs.com/package/react-validator-prop-types